common/package.json

```diff
@@ -56,6 +56,7 @@
     "nprogress": "^0.2.0",
     "openseadragon": "^2.4.0",
     "prismic-client-beta": "npm:@prismicio/client@beta",
+    "prismic-helpers-beta": "npm:@prismicio/helpers@beta",
```

Comment:
I wonder if this is the key line – this is a dependency of `prismic-client-beta`, but possibly the `common` and `content` apps were pulling in subtly different versions, and that was causing an issue somewhere.

Reply:
This is weird, as yarn is supposed to be able to resolve this sort of thing 🤷 - glad it's fixed!